fullstop-violation/fullstop-violation-jpa/src/main/java/org/zalando/stups/fullstop/rule/service/RuleEntityService.java

```diff
@@ -0,0 +1,17 @@
+package org.zalando.stups.fullstop.rule.service;
+
+import org.zalando.stups.fullstop.rule.entity.RuleDTO;
+import org.zalando.stups.fullstop.rule.entity.RuleEntity;
+
+import java.util.List;
+
+public interface RuleEntityService {
+
+    RuleEntity save(RuleDTO ruleDTO);
+
+    RuleEntity update(RuleDTO ruleDTO, Long id);
```

Comment:
interesting point from Jan: should we really "update" existing Rules, or should we better replace them (set expiryDate to yesterday and create a new one)
This way we would have a clean history -> Especially when we put the matched rule into the violation, it would be weird to allow later changes on the rule.